NitroxClient/GameLogic/Terrain.cs

```diff
@@ -33,31 +33,33 @@ public void CellLoaded(Int3 batchId, Int3 cellId, int level)
             LargeWorldStreamer.main.StartCoroutine(WaitAndAddCell(batchId, cellId, level));
             markCellsReadyForSync(0.5f);
         }
-        
+
         private IEnumerator WaitAndAddCell(Int3 batchId, Int3 cellId, int level)
         {
             yield return new WaitForSeconds(0.5f);
 
-            VisibleCell cell = new VisibleCell(batchId, cellId, level);
+            AbsoluteEntityCell cell = new AbsoluteEntityCell(batchId, cellId, level);
+            Log.Debug("Cell {0} loaded", cell);
```

Comment:
Should we leave these in?  They might be a little noisy.

Reply:
This one should definitely be removed, as it is extremely noisy.